tests/tests_app.py

```diff
@@ -286,7 +287,7 @@ def test_no_download_link_for_non_blueshirt(self) -> None:
         self.assertNotIn(download_url, html)
 
     def test_shows_download_link_for_blueshirt(self) -> None:
-        self.session.auth = ('blueshirt', 'blueshirt')
+        self.session.auth = (BLUESHIRT_SCOPE, BLUESHIRT_SCOPE)
```

Comment:
Not sure this is quite right? I think these happen to be the same string as the scope name, but aren't semantically the same.